azure-credits.html

```diff
@@ -18,7 +18,8 @@ <h1 class="h2">Azure credits for open source projects</h2>
         <div class="wrapper my-6 py-4">
             <div class="col-md-10 col-lg-7 mx-auto">
                 <h3 class="h4 font-weight-400 mb-4">
-                  This is some introductory text.
+                  We are offering Azure credits to open source projects for their use for testing, builds, and other infrustructure support needs. Projects must 
```

Comment:
Would be good to clarify (here or elsewhere) how long the aware lasts and if it is renewable.